tests/settings.py

```diff
@@ -90,6 +90,12 @@
     }
 ]
 
+DJANGO_LOCI_GEOCODER = 'ArcGIS'
+DJANGO_LOCI_GEOCODE_API_KEY = None
+DJANGO_LOCI_GEOCODE_STRICT_TEST = True
+DJANGO_LOCI_GEOCODE_DELAY = 1
+DJANGO_LOCI_GEOCODE_RETRIES = 3
+
```

Comment:
these look like the default values, I think you can remove them from here since they don't add any value